ray-operator/controllers/ray/common/pod.go

```diff
@@ -629,6 +629,13 @@ func setContainerEnvVars(pod *corev1.Pod, rayNodeType rayv1.RayNodeType, rayStar
 		usageEnv := corev1.EnvVar{Name: utils.RAY_USAGE_STATS_KUBERAY_IN_USE, Value: "1"}
 		container.Env = append(container.Env, usageEnv)
 	}
+	if rayNodeType == rayv1.HeadNode {
+		extraTagsEnv := corev1.EnvVar{
+			Name:  utils.RAY_USAGE_STATS_EXTRA_TAGS,
```

Comment:
Do we need to inject this env var to worker nodes?

Reply:
No, head node is enough.